docs/upgrade.html

```diff
@@ -79,7 +79,7 @@ <h5><a id="upgrade_360_zk" href="#upgrade_360_zk">Upgrading ZooKeeper-based clus
         </li>
     </ol>
 
-    <h5><a id="upgrade_360_kraft" href="#upgrade_360_kraft">Upgrading KRaft-based clusters</a></h5>
+    <h5><a id="upgrade_361_kraft" href="#upgrade_361_kraft">Upgrading KRaft-based clusters</a></h5>
```

Comment:
note this is still `upgrade_360_kraft` in kafka-site, which I think is inconsistent. So fixing it here